buildpack/centos8/Dockerfile

```diff
@@ -7,13 +7,13 @@ RUN yum -y install \
     wget
 
 RUN yum install -y yum-utils \
-    && dnf config-manager --enable PowerTools \
+    && dnf config-manager --enable powertools \
```

Comment:
Build also identified a failure for CentOS8 images:
```
Error: No matching repo to modify: PowerTools.
```
which is not related to this PR and probably some recent upstream changes, but I'll fix it anyway here to have the CI :green_circle: